.devcontainer/devcontainer.json

```diff
@@ -4,10 +4,12 @@
         "--cap-add=SYS_PTRACE",
         "--security-opt",
         "seccomp=unconfined",
-        "--network=host",
         "--privileged",
         "-v",
         "/dev/bus/usb:/dev/bus/usb:ro",
+        "-v",
+        // Change this and provide a path to your local FastModelsPortfolio_11.16
+        "/path/to/fmp/FastModelsPortfolio_11.16:/opt/FastModelsPortfolio_11.16:ro",
```

Comment:
I think it should be better to use a `mount`: https://code.visualstudio.com/remote/advancedcontainers/add-local-file-mount .